src/wing/CCPACSWingSection.h

```diff
@@ -37,32 +37,58 @@ class CCPACSWingSection : public generated::CPACSWingSection
 public:
     TIGL_EXPORT CCPACSWingSection(CCPACSWingSections* parent, CTiglUIDManager* uidMgr);
 
-    // Get element count for this section
+    /**
+    * @brief Get element count for this section
```

Comment:
We could mention here, that while CPACS allows for several elements per section, TiGL only has support for sections containing exactly one element.